third_party/parquet/parquet_types.h

```diff
@@ -0,0 +1,2130 @@
+/**
```

Comment:
Why do you need to check this in? I'm guessing bazel can't handle building thrift?

Reply:
Thanks @skye. Parquet needs a thrift type file. This file is generated automatically. This file is also a one time effort for each version and will not change when regenerate. However, it need both flex and bison to build. I tried to put flex and bison in but realized that it is quite complicated. That will adds more dependencies.

On the other hand, as this is just a one time effort of generated file, I think it makes sense to build the file and add the generated file.